cli/pack/app_dir.go

```diff
@@ -208,7 +208,14 @@ func generateVersionFile(appDirPath string, ctx *context.Ctx) error {
 
 	// Tarantool version
 	if ctx.Tarantool.TarantoolIsEnterprise {
-		tarantoolVersionFilePath := filepath.Join(ctx.Tarantool.TarantoolDir, "VERSION")
+		var tarantoolVersionFilePath string
+
+		if ctx.Build.InDocker {
+			tarantoolVersionFilePath = filepath.Join(ctx.Build.SDKPath, "VERSION")
```

Comment:
I see some copy-paste here. Let's store `"VERSION"` string in some variable, or better save to some `tarantoolVersionFileDir` variable a parent dir for VERSION file to call `Join` once.